src/server/scripts/Commands/cs_ahbot.cpp

```diff
@@ -193,7 +154,7 @@ class ahbot_commandscript : public CommandScript
             statusInfo[AUCTION_HOUSE_HORDE].ItemsCount +
             statusInfo[AUCTION_HOUSE_NEUTRAL].ItemsCount);
 
-        if (all)
+        if (all.has_value())
```

Comment:
```suggestion
        if (all)
```

Reply:
Yep. Remainder of my desperate attempt to fix the MSVC warning earlier...